cmp/grid/Grid.js

```diff
@@ -518,6 +519,17 @@ class GridLocalModel extends HoistModel {
         };
     }
 
+    sizingModeReaction() {
+        const {model} = this;
+        return {
+            track: () => model.sizingMode,
+            run: () => {
+                if (model.autosizeOptions.mode !== GridAutosizeMode.ON_SIZING_MODE) return;
+                model.autosizeAsync();
+            }
+        };
+    }
```

Comment:
Is it weird that the Grid component is setting up a reaction on a model property that calls only model methods? Why not have this setup on the model itself?

(Was also wondering if cleaner to only setup the reaction in the first place if we're tracking the overall mode, but I don't think there's really a material difference, so if this is more consistent I'm a-OK with it.)

Reply:
atm and I discussed this is ok